src/platform/tizenrt/iotjs_module_pwm-tizenrt.c

```diff
@@ -15,11 +15,12 @@
 
 #if defined(__TIZENRT__)
 
-#include <iotbus_error.h>
-#include <iotbus_pwm.h>
 #include <stdlib.h>
 #include <sys/ioctl.h>
 
+#include <iotbus_error.h>
```

Comment:
I'd vote for the order top-to-bottom (used by Google in their C/C++ codebase):
 - standard headers (like <stdio.h>, <sys/types.h> and so on) first,
 - then dependencies (<some_tizen_api.h>, ..) 
 - and local project last ("../include/common.h")
alphabetically sorted (a to z).

This way local redefinitions clash against more general (system- or project- scoped) definitions.